src/common.ts

```diff
@@ -0,0 +1,194 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as util from 'util';
+import * as types from './types/core';
+
+const logging = require('@google-cloud/logging');
+const mapValues = require('lodash.mapvalues');
+
+type Callback = (err: Error, apiResponse: {}) => void;
+
+// Map of npm output levels to Stackdriver Logging levels.
+const NPM_LEVEL_NAME_TO_CODE = {
+  error: 3,
+  warn: 4,
+  info: 6,
+  verbose: 7,
```

Comment:
At first glance it looks strange that the next three levels all correspond to 7.  It could be good to add a comment to explain why.

Reply:
i have no idea why =)
this is just copied directly from the old one